PWGHF/D2H/Tasks/taskBplusReduced.cxx

```diff
@@ -511,54 +555,73 @@ struct HfTaskBplusReduced {
         }
 
         hfRedCandBpLite(
+          // B+ - meson features
+          invMassBplus,
+          ptCandBplus,
+          candidate.eta(),
+          candidate.phi(),
+          hfHelper.yBplus(candidate),
+          candidate.cpa(),
+          candidate.cpaXY(),
           candidate.chi2PCA(),
           candidate.decayLength(),
           candidate.decayLengthXY(),
           candidate.decayLengthNormalised(),
           candidate.decayLengthXYNormalised(),
+          candidate.impactParameterProduct(),
+          candidate.maxNormalisedDeltaIP(),
+          candidateMlScoreSig,
+          candidate.isSelBplusToD0Pi(),
+          // D-meson features
           invMassD0,
           ptD0,
-          candidate.ptProng1(),
+          decLenD0,
+          decLenXyD0,
           candidate.impactParameter0(),
-          candidate.impactParameter1(),
-          candidate.impactParameterProduct(),
-          prong1.tpcNSigmaPi(),
-          prong1.tofNSigmaPi(),
+          candD0.ptProngMin(),
+          candD0.absEtaProngMin(),
+          candD0.itsNClsProngMin(),
+          candD0.tpcNClsCrossedRowsProngMin(),
+          candD0.tpcChi2NClProngMax(),
+          candD0.tpcNSigmaPiProng0(),
+          candD0.tofNSigmaPiProng0(),
+          candD0.tpcTofNSigmaPiProng0(),
+          candD0.tpcNSigmaKaProng1(),
+          candD0.tofNSigmaKaProng1(),
+          candD0.tpcTofNSigmaKaProng1(),
```

Comment:
In case you do not distinguish here between D0 and D0bar, you should add both NsigmaPi and NsigmaKa for both prongs. Alternatively (better in my opinion) you can always have pion in position 0 and kaon in position 1, but for this you should check if it is D0 (so B-) or D0bar (so B+) to put the correct ones (`NsigmaPiProng1`, `NsigmaKaProng0` in the first case and viceversa in the second case)

Reply:
Thanks for spotting this!